### PR TITLE
Add support for custom puml keyword (refs: #20)

### PR DESCRIPTION
**Pull request by legwinskij**

### CODE REVIEW

File: mkdocs_puml/plugin.py [hunk 1]
```diff
@@ -61,6 +59,11 @@ def on_config(self, config: Config) -> Config:
             Full config of the mkdocs
         """
         self.puml = PlantUML(self.config['puml_url'], num_workers=self.config['num_workers'])
+        self.div_class_name = self.config['puml_keyword']
```
Comment (MikhailKravets) [inline mkdocs_puml/plugin.py:62]: `self.div_class_name` should always be `puml`. This is the class we set in the resulting html tag. We can apply various css styles (or fulfill other manipulations of the tag) using this class name.

You need to create new class attribute, for example, `self.puml_keyword` and set the value of the config.

File: mkdocs_puml/plugin.py [hunk 2]
```diff
@@ -61,6 +59,11 @@ def on_config(self, config: Config) -> Config:
...
+
+        self.regex = re.compile(rf"```{self.div_class_name}(.+?)```", flags=re.DOTALL)
```
Comment (MikhailKravets) [inline mkdocs_puml/plugin.py:64]: Continuing above

use new `self.puml_keyword` instead of `self.div_class_name`

**Review by MikhailKravets — CHANGES_REQUESTED** [review]

**Comment by MikhailKravets** [comment]
Hi @legwinskij, please take a look. I've added few comments.

File: mkdocs_puml/plugin.py [hunk 3]
```diff
@@ -20,8 +20,6 @@ class PlantUMLPlugin(BasePlugin):
                     num_workers: 10
 
     Attributes:
-        div_class_name (str): the class that will be set to resulting <div> tag
-                              containing the diagram
```
[the remark this answers is not in the document]
Comment (legwinskij) [reply mkdocs_puml/plugin.py:24]: Thanks for feedback, I've misunderstood the concept, I will get to it :).

File: mkdocs_puml/plugin.py (hunk 1 above)
Comment (legwinskij) [reply mkdocs_puml/plugin.py:62]: Again misunderstanding on my end, thanks for clarification, I will rework it!

**Comment by MikhailKravets** [comment]
Good job @legwinskij 👍 Will be added to a new release!

**Pull request merged**